api/rf/samples.py

```diff
@@ -30,8 +30,31 @@ def get_samples(data_bytes, data_type) -> np.ndarray:
         raise ("Datatype " + data_type + " not implemented")
     return samples
 
+def get_bytes_per_iq_sample(data_type):
+    """ 
+    Get the number of bytes per I+Q sample.
 
-def get_sample_length_from_byte_lenth(byte_length: int, data_type: str):
+    Parameters
+    ----------
+    data_type : str
+        The data type of the bytes.
+
+    
+    Returns
+    -------
+    int
+        The number of bytes per I+Q sample.
+    """
+    if data_type == "cf32_le" or data_type == "cf32" or data_type == "cf32_be":
+        return 8
+    elif data_type == "ci16_le" or data_type == "ci16" or data_type == "ci16_be":
+        return 4
+    elif data_type == "ci8" or data_type == "i8":
+        return 2
+    else:
+        raise ValueError("Datatype " + data_type + " not implemented")
+
+def get_sample_length_from_byte_length(byte_length: int, data_type: str):
```

Comment:
the name of this function is a little ambiguous, but i dont think it has to be a function at all, its just dividing two numbers

Reply:
I did create this function it was on a branch where `get_bytes_per_iq_sample` did not exist so it might not make sense now anymore

Reply:
Removed and updated